doc/cephadm/services/iscsi.rst

```diff
@@ -72,9 +72,19 @@ The specification can then be applied using:
 
 See :ref:`orchestrator-cli-placement-spec` for details of the placement specification.
 
-See also: :ref:`orchestrator-cli-service-spec`.
+Configuring iSCSI client
+========================
+
+The containerized iscsi service can be used from any host by `configuring the
+iSCSI initiator (client)`_, which will use TCP/IP to send SCSI commands to
+the iSCSI target (gateway).
 
 Further Reading
 ===============
 
 * RBD: :ref:`ceph-iscsi`
+
+See also: :ref:`orchestrator-cli-service-spec`.
```

Comment:
Did you mean to move `Service Specification` link from `Deploying iSCSI` to `Further Reading` section?  This PR should just add `Configuring iSCSI client section` IMO.

Reply:
Now `Further Reading` header went missing ;)  See Sebastian's suggestion.